conf/openmetadata.yaml

```diff
@@ -145,7 +145,7 @@ authenticationConfiguration:
   provider: ${AUTHENTICATION_PROVIDER:-basic}
   # This will only be valid when provider type specified is customOidc
   providerName: ${CUSTOM_OIDC_AUTHENTICATION_PROVIDER_NAME:-""}
-  publicKeyUrls: ${AUTHENTICATION_PUBLIC_KEYS:-[http://localhost:8585/api/v1/config/jwks]}
+  publicKeyUrls: ${AUTHENTICATION_PUBLIC_KEYS:-[http://localhost:8585/api/v1/system/config/jwks]}
```

Comment:
I will need to take care of this on Helm Chart side as well with the next release! I will add an issue ticket supporting this PR for the same!